README.rst

```diff
@@ -31,7 +31,7 @@ assume you should be comfortable using the master branch.
 Installation
 ============
 
-The main dependencies for wflow are an installation of Python 3.6, and PCRaster 4.2+.
+The main dependencies for wflow are an installation of Python 3.6+, and PCRaster 4.2+.
```

Comment:
I removed this plus here: https://github.com/openstreams/wflow/commit/f2172f622ae34561d5ebcd4110fb48f0221748b8

Since PCRaster doesn't work with Python 3.7, neither can wflow.